test_data.rb

```diff
@@ -491,6 +491,21 @@ def test_main_station_hints
     end
   end
 
+  def test_normalised_code
+    STATIONS.each do |row|
+      if (row["country"] != 'GB' )
```

Comment:
We usually don't use parenthsis here in ruby.

Reply:
Done. Thanks